### PR TITLE
Add OnceThereIsAHill

### PR DESCRIPTION
**Pull request by gogit-yifei**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2444)

### CODE REVIEW

File: Chinese/OnceThereIsAHill/OnceThereIsAHill.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+从前有座山，山里有个庙，庙里有个老和尚，给小和尚讲故事。故事讲的是：从前有座山，山里
```
Comment (casey-collab) [inline Chinese/OnceThereIsAHill/OnceThereIsAHill.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by gogit-yifei** [comment]
Thanks

**Comment by gogit-yifei** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cake:

**Pull request merged**